src/components/window-shade-button.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+
+import * as css from "./window-shade.sass";
```

Comment:
I'd rather use the separate .sass file for every component. Then it's clear where to go when you edit a particular component. And it's also clear that you can remove .sass file if you remove React component.

I know that this case is a bit specific, as we use styles defined in `content-configuration`. But I'd still repeat some styling, as then it's easier to know which styles are applied where. Now I'm not sure which rules affect header button and which affect the rest of the window shade. I wouldn't worry about repetition, as I think it's a smaller problem (it could be nice to have color variables etc. in the future).

Reply:
Yep, agreed on all points. But this was intended for work in a refactor, captured in a later PR.